dropshot/tests/test_pagination.rs

```diff
@@ -227,7 +227,7 @@ async fn test_paginate_errors() {
     struct ErrorTestCase {
         path: String,
         message: &'static str,
-    };
+    }
```

Comment:
I'm curious what's the story behind this?  I would have expected rustfmt to always put the semicolon here or always leave it off, but presumably we were style-clean both before and after this change.

Reply:
My bad; I'll put this back! It's an artifact from my defaulting to nightly, where it'll become a warning.